app/models/webauthn_configuration.rb

```diff
@@ -39,4 +43,11 @@ def self.selection_presenters(set)
       []
     end
   end
+
+  private
+
+  def valid_transports
+    return if transports.blank? || transports.difference(VALID_TRANSPORTS).blank?
+    errors.add(:transports, 'invalid transports', type: :invalid_transports)
```

Comment:
gotta translate this right?

Reply:
Not sure! Admittedly I was following some prior art here for custom errors at the model validation layer, e.g.

https://github.com/18F/identity-idp/blob/79d9299abdd300ff966e5479ec5f8164715ee2d5/app/models/agreements/iaa_gtc.rb#L34

https://github.com/18F/identity-idp/blob/79d9299abdd300ff966e5479ec5f8164715ee2d5/app/models/agreements/integration_usage.rb#L25-L28

In practice, it should never get this far, and the actual user-facing message would be generated by `WebauthnSetupForm`, which would certainly need to be translated (even if that's pretty unlikely to be seen in the real-world either).

Reply:
those models are basically offline reporting models, the creation is 100% on the backend and never user-facing, but this form is user-facing right?

Reply:
Yeah, I guess I assumed these would never end up being shown to a user, but there's nothing to say they couldn't. I can at least put some generic message here, similar to what's used in the WebauthnSetupForm class.

Reply:
Localized in 68a0fc396b0bb3aa626cead413d70be906142a37.